target/linux/ramips/image/mt7621.mk

```diff
@@ -361,6 +361,16 @@ define Device/iodata_wnpr2600g
 endef
 TARGET_DEVICES += iodata_wnpr2600g
 
+define Device/iptime_a8004t
+  MTK_SOC := mt7621
```

Comment:
Rename to "SOC" due to 58e71e322bc0e0edd08157075e9022c15ae33ec5.

Reply:
> > Does the device have a label with a MAC address on it?
> 
> AFAIK it does not have a label MAC sticker, still it needs to be confirmed by @perillamint.
> 
> > Why do you need wpad-basic?
> 
> mt7621 does not have any built-in wmac, so wpad-basic is not included by default.
> All other ramips SoC targets have wpad-mini or wpad-basic in DEFAULT_PACKAGES.

It does not have MAC address sticker either. It seems EFM networks do not like sticking MAC address stickers on their devices.